### ISSUE TITLE
Workspace navigation with shortcuts

### ISSUE THREAD

**Issue by RingOfStorms** [lilyinstarlight/nixos-cosmic#401]
I am not sure if I should report this here or on real cosmic since this is my entry for using cosmic.

When I have "Workspaces -> Displays Have Separate Workspaces" (the default) enabled and have multiple monitors:
- I cannot use super + j/k/ip/down to go up a workspace. Instead my cursor focuses on my other monitor and I am unable to go up a workspace via keyboard (I have to use workspace menu now).
- I can still go down just not up

Workaround for now: Select "Workspaces Span Displays", now going up and down is how I expect it to work with super + up/down (but not great if you don't want this behavior)

**Comment by RingOfStorms** [comment]
Okay I think I just don't understand where or how to change this shortcut. It is the ctrl + super + up/down that works. I want to remove ctrl part and replace the focus window ones. I've tried to clear the existing shortcuts and add to "Focus next workspace" but adding shortcuts simply does nothing, no feed back at all.

![image](https://github.com/user-attachments/assets/ba58eb91-c993-480e-9331-f363257cd36c)
![image](https://github.com/user-attachments/assets/1edf2049-76dc-4b0e-948c-5153c20e148d)
It always clears itself even if I "set" them

**Comment by RingOfStorms** [comment]
I also can't add custom keyboard shortcuts, the add keybinding button does nothing

![image](https://github.com/user-attachments/assets/d7c158bb-0983-4087-b3df-75ac70c4916c)

**Comment by RingOfStorms** [comment]
Alright I gave up on the UI but was able to get everything working manually editing the custom.ron file for keyboard shortcuts.

I also had to look at cosmic source to find the list of all possible "actions" since I didn't know how to move workspaces:
https://github.com/pop-os/cosmic-settings/blob/master/cosmic-settings/src/pages/input/keyboard/shortcuts/mod.rs#L460-L550

Example of a custom file:
https://git.joshuabell.xyz/dotfiles/~files/master/modules/de/cosmic/config/com.system76.CosmicSettings.Shortcuts/v1/custom

```
{
    (
        modifiers: [
            Super,
        ],
        key: "l",
    ): Focus(Right),
    (
        modifiers: [
            Super,
        ],
        key: "j",
    ): NextWorkspace,
    (
        modifiers: [
            Super,
        ],
        key: "h",
    ): Focus(Left),
    (
        modifiers: [
            Super,
        ],
        key: "k",
    ): PreviousWorkspace,
    (
        modifiers: [
            Super,
        ],
        key: "Return",
    ): Spawn("alacritty"),
    (
        modifiers: [
            Super,
        ],
        key: "space",
        description: Some("launcher"),
    ): System(Launcher),
    (
        modifiers: [
            Super,
        ],
    ): System(WorkspaceOverview),
    (
        modifiers: [
            Super,
        ],
        key: "Right",
    ): Disable,
    (
        modifiers: [
            Super,
        ],
        key: "Down",
    ): Disable,
    (
        modifiers: [
            Super,
        ],
        key: "Left",
    ): Disable,
    (
        modifiers: [
            Super,
        ],
        key: "Up",
    ): Disable,

}
```

**Comment by Elias-Graf** [comment]
I haven't read all the comments exactly, so correct me if I'm wrong, but this sounds very much like something that should be reported in the COSMIC repository.

**Comment by RingOfStorms** [comment]
Yeah I can close this out, made this at the start of my cosmic journey didn't know where the issue was originating from but you are right
